canon.make

```diff
@@ -1,102 +0,0 @@
-# Force updates if images are older than this. Should be updated for breaking changes to images.
```

Comment:
Just confirming we want to delete this? I noticed it's still [in rdk](https://github.com/viamrobotics/rdk/blob/cc7f18eecfccb321372b373f045500d0f6a8308d/canon.make)

Reply:
I've yet to remove the old-style (makefile-based) canon tooling from RDK. That'll be removed later this week.

Reply:
ty!